lib/config.py

```diff
@@ -36,6 +36,15 @@
         'username': 'OpenStack user name with Swift privileges',
         'password': 'OpenStack user password'
     },
+    'credhub': {
```

Comment:
So, it is not mandatory parameter now?

Reply:
same as replied [here](https://github.com/cloudfoundry-incubator/service-fabrik-backup-restore/pull/35#discussion_r197322904)